src/java/yummy/YummyConstructor.java

```diff
@@ -19,6 +20,9 @@
 
 
 public class YummyConstructor extends SafeConstructor {
+    public YummyConstructor() {
+        super(new LoaderOptions());
```

Comment:
`LoaderOptions` is a manatory arg for [SafeConstructor](https://javadoc.io/doc/org.yaml/snakeyaml/1.33/org/yaml/snakeyaml/constructor/SafeConstructor.html) since `snakeyaml 2.0`